### ISSUE TITLE
FTB Genesis: A lot of crashes since last update, potential MineTogether issue?

### ISSUE THREAD

**Issue by atealein** [FTBTeam/FTB-Modpack-Issues#4351]
### Modpack

FTB Genesis

### Modpack version

1.3.1

### Log Files

[crash-2023-11-26_12.09.05-client.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/13466745/crash-2023-11-26_12.09.05-client.txt)
[crash-2023-11-26_13.02.15-client.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/13466746/crash-2023-11-26_13.02.15-client.txt)
[crash-2023-11-26_13.33.07-client.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/13466747/crash-2023-11-26_13.33.07-client.txt)

### Describe the bug

Happens at random moments, the entire instance crashes. I don't use MineTogether and I have disabled the buttons and showing up in chat from the title screen options. This wasn't an issue in previous updates.

### Steps to reproduce

I am not sure how to reproduce it, seems random right now, but I have supplied the three crash logs I have

### Expected behaviour

Not get crash from mod I am not using.

**Comment by slimpie2003** [comment]
i had the same problem. i fixed it by simply disabling MineTogether in the mods tab in the launcher.

**Comment by Saereth** [comment]
https://github.com/CreeperHost/MineTogether/issues/758

**Comment by atealein** [comment]
I can confirm that in the new update 1.3.2 the issue is still there and it seems to be triggered when you go to sleep (there are other triggers, i have crashed away from bed as well, but i have had few people tell me they also experienced crashing when trying to sleep).

**Comment by slimpie2003** [comment]
> I can confirm that in the new update 1.3.2 the issue is still there and it seems to be triggered when you go to sleep (there are other triggers, i have crashed away from bed as well, but i have had few people tell me they also experienced crashing when trying to sleep).

yes! this was the most constant trigger for me as wel tho opening an inventory would also often cause the crash.

**Comment by Saereth** [comment]
yep this was reported and has now been fixed for the update releasing soon
